python-sdk/tests/benchmark/constants.py

```diff
@@ -0,0 +1,13 @@
+import os
```

Comment:
The convention we used for the Python SDK is to have a `settings.py` to retrieve things from environment variables and a `constants.py` file to static things. I suggest we name this `settings.py`.